cartoframes/data/enrichment/enrichment_service.py

```diff
@@ -96,6 +117,26 @@ def __copy_data_and_generate_enrichment_id(data, enrichment_id_column, geometry_
     return data_copy
 
 
+def __process_variables(variables):
+
+    variables_result = list()
+    if isinstance(variables, Variable):
+        variables_result = [variables]
+    elif isinstance(variables, str):
+        variables_result = [Variable.get(variables)]
+    elif isinstance(variables, list):
+        first_element = variables[0]
+
+        if isinstance(first_element, str):
+            variables_result = [Variable.get(variable) for variable in variables]
```

Comment:
This can run multiple queries to our SQL API, besides being slow we'll have problems with rate limits.

I'd add a method `Variable.list([variables])` in the catalog instead, that makes a unique query.

Reply:
You are right, I didn't know that method!

Reply:
Well, actually it does not exist :) but we should create it. We'll save many queries via SQL API which will add a lot of latency and rate limits problems

I guess we can add a `list` method to the `CatalogEntity` that receives a list of `id` and makes a single request to the SQL API.

What do you think @esloho ?

Reply:
I would create another issue to improve that method and would merge this PR in order to keep advancing!

Reply:
Refer to https://github.com/CartoDB/data-observatory/issues/237

Reply:
I just saw this, sorry!

Yes, I think it would be useful to have such method although I agree with @alejandrohall that could be better to address it in a separated issue. 

It actually goes on well with an idea I have about filters (I was thinking on supporting different SQL filters like the "x in list" and not only the current "category_id = x").